### PR TITLE
e2e react-host-remote

### PR DESCRIPTION
**Pull request by Andranik-mikhaielian**
Screenshot 2022-12-23 at 12 54 49 PM
Screenshot 2022-12-23 at 12 55 02 PM

### CODE REVIEW

File: cypress/fixtures/constants.ts [hunk 1]
```diff
@@ -457,7 +463,7 @@ export class Constants {
         nextjsSsrDocumentationUrl: 'https://nextjs.org/docs',
         nextjsSsrLearnUrl: 'https://nextjs.org/learn/foundations/about-nextjs',
         nextjsSsrExamplesUrl: 'https://github.com/vercel/next.js/tree/deprecated-main/examples',
-        nativeFederationReactUrl: 'https://reactjs.org',
```
Comment (edchai) [inline cypress/fixtures/constants.ts]: please be careful with deleting. I used this link in one of my samples. revert deletion

File: cypress/fixtures/constants.ts [hunk 2]
```diff
@@ -481,9 +487,4 @@ export class Constants {
         cpSharedComponentFile: 'cp server-side-render-only/e2e/fixtures/SharedComponent.js server-side-render-only/remoteServer',
         cpOriginalSharedComponentFile: 'cp server-side-render-only/e2e/fixtures/originalFile/SharedComponent.js server-side-render-only/remoteServer'
     }
-
-    public static readonly fieldsNames = {
```
Comment (edchai) [inline cypress/fixtures/constants.ts]: why deleted?

File: react-nextjs/react-host-remote/e2e/CommonCheck.cy.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h2,
+                text: property.appButtonExpensesH2
+            })
+        }
+
+    })
+})
+    }
```
Comment (edchai) [inline react-nextjs/react-host-remote/e2e/CommonCheck.cy.ts]: some strange bracket position. I suppose you have some bracket error somewhere

File: cypress/fixtures/constants.ts [hunk 4]
```diff
@@ -283,6 +283,12 @@ export class Constants {
             host: 'Host button: ',
             remote: 'Remote button: '
         },
+
+        reactHostRemoteContainerHeader: 'This is the container App hosted at localhost:8080',
+        reactHostRemoteContainerDiv: 'This component is from the Host React App hosted at localhost:8081',
+        reactHostRemoteHostedDiv: 'This is the Remote App hosted at localhost:8081)',
+        reactHostRemoteHostedInvoices: 'Invoices',
```
Comment (edchai) [inline cypress/fixtures/constants.ts]: merge Invoices and Expenses consts into one object with named variables, as you can see in example ended on 285 string

**Review by edchai — CHANGES_REQUESTED** [review]

File: cypress/fixtures/constants.ts (hunk 1 above)
Comment (Andranik-mikhaielian) [reply cypress/fixtures/constants.ts]: thank you. reverted

File: cypress/fixtures/constants.ts (hunk 2 above)
Comment (Andranik-mikhaielian) [reply cypress/fixtures/constants.ts]: reverted, not sure how it's happened.

File: cypress/fixtures/constants.ts (hunk 4 above)
Comment (Andranik-mikhaielian) [reply cypress/fixtures/constants.ts]: fixed

File: react-nextjs/react-host-remote/e2e/CommonCheck.cy.ts (hunk 3 above)
Comment (Andranik-mikhaielian) [reply react-nextjs/react-host-remote/e2e/CommonCheck.cy.ts]: used reformatCode, command+option+l
please check

**Review by edchai — APPROVED** [review]

**Review by edchai — APPROVED** [review]

**Pull request merged**